docs/source/components/nodes/stereo_depth.rst

```diff
@@ -180,6 +136,188 @@ Currently configurable blocks
+
+Applying both of those options is possible, which would set the minimum depth to 1/4 of the standard settings, but at such short distances the minimum depth is limited by focal length, which is 19.6cm, since OAK-D mono cameras have fixed focus distance: 19.6cm - infinity.
+
+See `these examples <https://github.com/luxonis/depthai-experiments/tree/master/gen2-camera-demo#real-time-depth-from-depthai-stereo-pair>`__ for how to enable Extended Disparity.
+
+Max stereo depth distance
+#########################
+
+The maximum depth perception distance depends on the :ref:`accuracy of the depth perception <Depth perception accuracy>`. The formula used to calculate this distance is an approximation, but is as follows:
+
+.. code-block:: python
+
+  Dm = (baseline/2) * tan((90 - HFOV / HPixels)*pi/180)
+
+So using this formula for existing models the *theoretical* max distance is:
+
+.. code-block:: python
+
+  # For OAK-D (7.5cm baseline)
+  Dm = (7.5/2) * tan((90 - 71.9/1280)*pi/180) = 3825.03cm = 38.25 meters
+  
+  # For OAK-D-CM4 (9cm baseline)
+  Dm = (9/2) * tan((90 - 71.9/1280)*pi/180) = 4590.04cm = 45.9 meters
+
+If greater precision for long range measurements is required, consider enabling Subpixel Disparity or using a larger baseline distance between mono cameras. For a custom baseline, you could consider using `OAK-FFC <https://docs.luxonis.com/projects/hardware/en/latest/pages/DM1090.html>`__ device or design your own baseboard PCB with required baseline. For more information see Subpixel Disparity under the Stereo Mode tab in :ref:`this <Currently configurable blocks>` table.
+
+Depth perception accuracy
+#########################
+
+Disparity depth works by matching features from one image to the other and its accuracy is based on multiple parameters:
+
+* Texture of objects / backgrounds
+
+Backgrounds may interfere with the object detection, since backgrounds are objects too, which will make depth perception less accurate. So disparity depth works very well outdoors as there are very rarely perfectly-clean/blank surfaces there - but these are relatively commonplace indoors (in clean buildings at least).
+
+* Lighting
+
+If the illumination is low, the diparity map will be of low confidence, which will result in a noisy depth map.
+
+* Baseline / distance to objects
+
+Lower baseline leads to less accurate depth perception, which also means accuracy decreases as distance to objects increases. This is due to the fact that when an object is further away, the distance between mono cameras (baseline) becomes negligible. This is essentially the same as if the two cameras were moved closer and closer together, until only one camera would remain. At that point baseline would be equal to 0 and no depth information would be available.
```

Comment:
`Lower baseline leads to less accurate depth perception` - > Lower baseline enables us to detect the depth at more closer distance as long as the object is visible in both the frames. However this reduces the accuracy for large distance due to less pixels representing the object and the disparity representing the further distance is too small. 
So the common norm is to adjust the baseline based on how far/closer distance we would be interested in measuring.

Can you paraphrase this properly and add it here..